apps/lib/apps.c

```diff
@@ -2307,7 +2307,13 @@ int do_X509_verify(X509 *x, EVP_PKEY *pkey, STACK_OF(OPENSSL_STRING) *vfyopts)
     int rv = 0;
 
     if (do_x509_init(x, vfyopts) > 0)
-        rv = (X509_verify(x, pkey) > 0);
+        /* 
+         * X509_verify returns 1 if the signature is valid, 
+         * 0 if the signature check fails, or -1 if error occurs. 
+         */
```

Comment:
I think you've misunderstood my comment. I would like to document the return value of do_X509_verify()